src/main/resources/templates/aeolus/python/default_static.sh

```
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+set -e
+export AEOLUS_INITIAL_DIRECTORY=${PWD}
+static_code_analysis () {
+  echo '⚙️ executing static_code_analysis'
+  ruff check --config=ruff-student.toml --output-format=sarif --output-file=ruff.sarif --exit-zero "${studentParentWorkingDirectoryName}"
```

Shellcheck (0.10.0)

[warning] studentParentWorkingDirectoryName is referenced but not assigned.

(SC2154)